web/themes/new_weather_theme/assets/sass/components/hourly-table.scss

```diff
@@ -15,31 +15,40 @@
 
 /* Hourly forecast table overrides
 ------------------------------------- */
-.usa-table.hourly-table {
-  td {
-    min-width: units("card");
-    padding-right: units(2);
-    white-space: normal;
-  }
+$expand-duration: 250ms; 
 
-  // th[scope="row"] {
-  //   white-space: nowrap;
-  // }
+.expand-transition {
+  @include at-media("tablet-lg") {
+    transition: width ease-in $expand-duration, max-width ease-in $expand-duration, margin-left ease-in $expand-duration, margin-right ease-in $expand-duration,;
 
-  th {
-    padding-right: units(1.5);
   }
 
+  @media (prefers-reduced-motion) {
+    transition: none;
+  }
+}
```

Comment:
Nothing to change here, just a question: would it be good practice to either namespace `.expand-transition` just to the hourly table, or alternatively move it to a more general file if we might use it elsewhere? I can imagine a future where someone wants to update it and struggles to find it. (Though really, with a good IDE, they should be able to search for it.)

Reply:
That's a great point, I think this one is pretty specific to this component so I add the `hourly_table` namespace as a prefix.